### PR TITLE
First pass - adds the grid, on /angelfish only

### PR DESCRIPTION
**Pull request by andycarrell**
- Adds the grid, with items, and gap etc
- Updates progress with a _really_ little version

To do (in future PRs)
- Update typography (everything a little smaller)
- Figure out how to not have the tick in this case, and still have it on the old screen

### CODE REVIEW

File: app/components/Progress/Progress.tsx [hunk 1]
```diff
@@ -26,23 +27,13 @@ export const Progress = ({
...
         className={styles["progress-bar"]}
         style={{ width: `${secondDose * 100}%` }}
       >
-        <div
-          className={cx(styles["progress-second"], "animate-grow-width", {
-            [styles["progress-bar-small"]]: size === "small",
-            [styles["progress-bar-large"]]: size === "large",
-          })}
-        />
+        <div className={cx(styles["progress-second"], "animate-grow-width")} />
```
Comment (andycarrell) [inline app/components/Progress/Progress.tsx:36]: We don't actually need these! the parent border-radius keeps everything toight

File: app/pages/index.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
         </div>
       </section>
       <section>
-        <div className="page-container mx-5 sm:mx-auto">
+        <div className="max-w-[600px] mx-5 sm:mx-auto">
```
Comment (andycarrell) [inline app/pages/index.tsx:79]: Reverted this so I can change the container classname on the new page

File: app/styles/globals.scss [hunk 3]
```diff
@@ -27,13 +28,34 @@ body {
...
+    background: var(--background-tint);
+    border-radius: $radius-large;
+
+    grid-column: span 12 / span 12;
+
+    @media (min-width: 640px) {
+      grid-column: span 6 / span 6;
+    }
+
+    @media (min-width: 1024px) {
```
Comment (andycarrell) [inline app/styles/globals.scss:53]: Will extract these into breakpoints 🙏 - might do in a separate PR, because it would be good to have a wider convo about what break points we have

File: app/styles/globals.scss [hunk 4]
```diff
@@ -27,13 +28,34 @@ body {
...
 }
 
 .container-data {
   background: var(--background-tint);
 }
 
+.doses-data-grid {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  gap: 1.25rem;
```
Comment (andycarrell) [inline app/styles/globals.scss:41]: @finnclark - this is a gap! Will use one of your space vars after your PR is merged

**Review by finnclark — APPROVED** [review]
looks fucken sick bruv
